tests/collector.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
I assume you're waiting for https://github.com/metarhia/MetaSync/pull/87 to land in order to add this to `npm test`, right?